packages/toolkit/src/tests/createSlice.test.ts

```diff
@@ -191,7 +191,8 @@ describe('createSlice', () => {
                 .addCase('increment', (state) => state + 1)
                 .addCase('increment', (state) => state + 1),
           })
-        ).toThrowErrorMatchingInlineSnapshot(
+          slice.reducer(undefined, { type: 'unrelated' })
```

Comment:
This is the only downside to this I could find. As the builder is now called lazily on first reducer call, `createSlice` will not immediately toggle these errors.
But they will be triggered immediately on adding this to a store, so I'm pretty sure that would be okay.